### ISSUE TITLE
Skeb uploads are broken

### ISSUE THREAD

**Issue by 9suika** [danbooru/danbooru#5773]
Looks like request keys are now set from JS.

**Comment by hdk5** [comment]
won't make a merge request, but here's what we did at our booru, in case that helps:

```rb
# app/logical/source/extractor/skeb.rb:60
        if response.status == 429 && response.mime_type == "text/html"
          request_key = response.to_s.match(/request_key=(\h{8}:\h{8})/)&.[](1)
          if request_key.present?
            self.cached_request_key = request_key
            response = http.cache(1.minute).get(api_url)
          end
        end
```

**Comment by nonamethanks** [comment]
Should be fixed by #5795.
